models/tensorflow/Lenet5.py

```diff
@@ -1,5 +1,5 @@
 class Lenet5:
-    def get_model():
+    def get_model(self):
```

Comment:
Remove self. This is suppose to be treated as a class method. We invoke this using `Lenet5.get_model()`.